### DISCUSSION TITLE
"Secret Crush": How to enable people to see who liked them

### DISCUSSION THREAD

**Discussion by ethanblake4** [PullDating/Org#77] [Ideas & Features]
If we're going to make it so that people can just see who liked them directly with no blur, it's worth looking at other apps. The only mainstream dating app that does this, afaik, is Facebook Dating. And there's actually a good reason: swiping on someone you know IRL. Most dating apps would have a bit of a problem with this if you could see likes, but FB dating solves it the way only Facebook could do, by automatically filtering out anyone in your Facebook friends list - unless you mark that you have a crush on  a specific person. That person can't see that you did it, but if they also mark a crush on you, you're then able to swipe each other.

Obviously we can't really do this (unless we required FB login) but since I assume we're not doing superlikes, what if the 'swipe up' action did a 'secret crush' where you'd like someone without showing up in their likes list? Obviously this would have to be limited to a few per day or else some people would just always do it.

**Comment by ajbott** [comment]
Interesting problem to solve. I think the contact hiding feature would interfere with secret crust feature. There must be a way to allow people that know each other IRL to match with each other on the app if they both like each other already.

**Reply by conaman551** [reply]
Not sure what the issue is with liking a person you already know...
